### PR TITLE
PR Review and README

### PR DESCRIPTION
**Pull request by jvoll**
I did a quick QA pass/PR review on what was in `master` and filled in our README. After this lands I'd like to get this published.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,40 @@ be used for all new documentation going forward. It currently is used for the
...
+# Developing
+The theme itself is not "runnable". The only way to do development on the theme
+and be able to quickly test your changes is by using it in another project via
+`npm link`.
+
+```bash
+git clone git@github.com:mobify/documentation-theme.git
+cd documentation-theme
+npm i
+npm link
```
Comment (gsaray) [inline README.md]: this should be `npm link @mobify/documentation-theme`

File: README.md [hunk 2]
```diff
@@ -3,13 +3,40 @@ be used for all new documentation going forward. It currently is used for the
...
+cd ../<project-that-uses-theme>
+npm link @mobify/documentation-theme
```
Comment (gsaray) [inline README.md]: this should be `npm link documentation-theme`

**Comment by gsaray** [comment]
Ignore my comments, your instruction works with node 8. 👍

**Comment by jvoll** [comment]
A few more changes are incoming to this PR. We discovered that renaming the package requires some further renaming. I'll tackle that tomorrow.

**Pull request merged**